scripts/main/photo.js

```diff
@@ -344,31 +399,27 @@ photo.setTitle = function (photoIDs) {
 
 		basicModal.close();
 
-		let newTitle = data.title;
+		const newTitle = data.title ? data.title : null;
 
 		if (visible.photo()) {
-			photo.json.title = newTitle === "" ? "Untitled" : newTitle;
+			// TODO: The JSON should be left untouched. Replacing `null` by `lychee.locale["UNTITLED"]` should happen on the GUI layer, i.e. somewhere inside the `view` component
+			photo.json.title = newTitle ? newTitle : lychee.locale["UNTITLED"];
 			view.photo.title();
 		}
 
 		photoIDs.forEach(function (id) {
+			// TODO: The line below looks suspicious: It is inconsistent to the code some lines above.
 			album.getByID(id).title = newTitle;
```

Comment:
> The JSON should be left untouched, i.e. exactly represent the object as it has been received from the backend, and an empty title should be replaced by the localized variant of "Untitled" on the GUI layer.

Yes, I agree about that. But it may not be worth it to fix it.

Reply:
Fixed in: https://github.com/LycheeOrg/Lychee-front/pull/282/commits/f6408a2bca857b8777db8fb324d71433b4fd296a